k8s/demo/cassandra/README.md

```diff
@@ -21,35 +20,35 @@ Whether you use OpenEBS Local PV or cStor, you can set up the Kubernetes cluster
 1. Install OpenEBS
 2. Select OpenEBS storage engine
 3. Configure OpenEBS LocalPV StorageClass
-4. Install Kudo operator to install Cassandra
+4. Install Kudo operator
 5. Install Kudo based Cassandra
 6. Verify Cassandra is up and running
-7. Testing Cassandra Performance on OpenEBS
+7. Testing Cassandra performance on OpenEBS
 
 ### Install OpenEBS
 
-If OpenEBS is not installed in your K8s cluster, this can be done from [here](https://docs.openebs.io/docs/next/overview.html). If OpenEBS is already installed, go to the next step.
+If OpenEBS is not installed in your K8s cluster, this can be done from [here](https://docs.openebs.io/docs/next/overview.html). If OpenEBS is already installed, go to next step.
```

Comment:
```suggestion
If OpenEBS is not installed in your K8s cluster, this can be done from [here](https://docs.openebs.io/docs/next/overview.html). If OpenEBS is already installed, go to the next step.
```